### DISCUSSION TITLE
Certain Placeholder Requirement for Rankup in CMI Ranks

### DISCUSSION THREAD

**Discussion by MrStalinko** [Zrips/CMI#5157] [Q&A]
Hey! I was wondering if there is a way to reference a placeholder as a requirement to rank up in CMI Ranks. For example, to rankup:

RankName:
  Enabled: true
  DisplayName: '{#e05a00}RankName'
  DefaultRank: false
  AutoRankup: false
  RankupConfirmation: false
  NextRanks:
  - Last
  MoneyCost: 0
  ExpCost: 0
  Commands:
  - broadcast! &6[playerDisplayName] &eleveled up to RankName rank!
  - lp user [playername] promote ranks
  StatsRequirements:
  - travel:1000
  - FromLastDeath:10
  **- Placeholder == true**
  ItemRequirement:
  - cobblestone:1

Basically I'm trying to increase the pool of things we can have as a condition to rankup. I know that's not exactly how it works but does CMI Ranks have the ability to do this somehow and how would I accomplish that? Thanks!

**Comment by CloudeLecaw** [comment]
no thats not possible you see what is possible here:
https://www.zrips.net/cmi/rankup-system/
